.travis.yml

```diff
@@ -6,7 +6,7 @@ jdk:
 env:
   matrix:
   - CI_TEST: ci-fast
-    CI_SCALA_VERSION: 2.11.11
+    CI_SCALA_VERSION: 2.12.3
     CI_PUBLISH: true
   - CI_TEST: ci-fast
     CI_SCALA_VERSION: 2.12.3
```

Comment:
This entry here runs tests in 2.12 but no publish, we should run ci_publish here